integration/mediation-tests/tests-sample/src/test/java/org/wso2/carbon/esb/samples/test/mediation/cache/Sample420TestCase.java

```diff
@@ -55,21 +56,15 @@ public void testSimpleCachingNotExists() throws AxisFault, XPathExpressionExcept
 
             response = axis2Client.sendSimpleStockQuoteRequest(getMainSequenceURL(), "", "IBM");
 
-            if (!firstResponse.equalsIgnoreCase(response.getFirstElement().toString())) {
-                assertFalse(false, "Caching is less than 20 seconds");
-            }
+            assertEquals(firstResponse, response.getFirstElement().toString(), "Caching is less than 20 seconds");
 
             Thread.sleep(2000);
```

Comment:
Why do we need a Thread.sleep(2000)?

Reply:
The timeout I have set in the Sample is 20s. The test here is to check if the cache is evicted after 20s. I have added a comment for this in PR #1167